spec/features/enter_name_spec.rb

```diff
@@ -0,0 +1,9 @@
+feature 'Player can enter name' do
+  scenario 'Player can enter name and see it on screen' do
+    visit('/')
+    page.fill_in("player_name", with: 'Kirsty')
+    page.click_button('Start')
+    expect(page).to have_current_path('/name')
+    expect(page).to have_selector('#player_name', text: 'Kirsty')
+  end
+end
```

Comment:
I like how you use the have_selector method to check an ID on the page contains a certain text field